Sources/EpoxyBars/BarInstaller/BarContainer.swift

```diff
@@ -208,6 +213,12 @@ extension InternalBarContainer {
       "The view controller's view must not be a scroll view. Nest any scroll views in a container.")
   }
 
+  /// Sets the `layoutMargin` corresponding to the container's `position`
+  func setLayoutMargin(_ margin: CGFloat) {
+    guard insetMargins else { return }
+    layoutMargins[keyPath: position.inset] = margin
```

Comment:
Don't we need some logic to handle when `insetMargins` goes from `true` to `false` to clear out the old margins?
```suggestion
    layoutMargins[keyPath: position.inset] = insetMargins ? margin : 0
```

Reply:
Good point! Will add a test case that covers this as well